build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    extra["compileVersion"] = 34
+    extra["compileVersion"] = 35
```

Comment:
I had this locally and it's actually needed for this to compile... I extracted this change to a separate PR: #2525